src/Promitor.Integrations.AzureStorageQueue/AzureStorageQueue.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Auth;
+using Microsoft.WindowsAzure.Storage.Queue;
+
+namespace Promitor.Integrations.AzureStorageQueue
+{
+    public class AzureStorageQueue
```

Comment:
Let's rename this to `AzureStorageQueueClient` to align with `AzureMonitorClient`